coco-ssd/src/index.ts

```diff
@@ -102,11 +102,14 @@ export class ObjectDetection {
    * @param maxNumBoxes The maximum number of bounding boxes of detected
    * objects. There can be multiple objects of the same class, but at different
    * locations. Defaults to 20.
+   * @param minScore The minimum score of the returned bounding boxes of detected objects.
```

Comment:
Sorry one more change, This line is also over 80 characters.

Reply:
My fault  😄